### PR TITLE
True Neutral abolishment

### PR DESCRIPTION
**Pull request by Danarca**

### CODE REVIEW

File: common/decisions/imperia_true_neutral_status.txt [hunk 1]
```diff
@@ -108,3 +108,51 @@ consider_true_neutral_status = {
...
 	}
 }
+
+abolish_true_neutral = {
+	is_shown = {
+		has_modifier = imperia_true_neutral
+		# This variable is active while the event is active
+		NOT = {
+			has_variable = considering_abolishing_neutrality
+		}
```
Comment (HiddeLekanne) [inline common/decisions/imperia_true_neutral_status.txt]: better to do possible I think

File: localization/english/imperia_true_neutral_status_l_english.yml [hunk 2]
```diff
@@ -1,16 +1,24 @@
 ﻿l_english:
  ask_for_true_neutral_status: "Petition for True Neutral Status"
  ask_for_true_neutral_status_desc: "We should ask our neighbours and the Great Powers to cement our status as a neutral country in International law."
+
  consider_true_neutral_status: "Consider a Neutrality petition"
  consider_true_neutral_status_desc: "There are treaties that needs to be signed... Should we get to it?"
+
+ if_you_accept_tt: "If you accept:"
+ abolish_true_neutral: "Abolish Neutrality"
+ abolish_true_neutral_desc: "We no longer wish to be a Neutral power. No one will be happy about this."
```
Comment (HiddeLekanne) [inline localization/english/imperia_true_neutral_status_l_english.yml]: The powers that be and our neighbors.

**Review by HiddeLekanne — CHANGES_REQUESTED** [review]

File: common/decisions/imperia_true_neutral_status.txt (hunk 1 above)
Comment (Danarca) [reply common/decisions/imperia_true_neutral_status.txt]: image

File: localization/english/imperia_true_neutral_status_l_english.yml (hunk 2 above)
Comment (Danarca) [reply localization/english/imperia_true_neutral_status_l_english.yml]: See other comments pic :)

**Review by HiddeLekanne — APPROVED** [review]

**Pull request merged**